src/instructlab/model/backends/backends.py

```diff
@@ -201,19 +201,21 @@ def get(model_path: pathlib.Path, backend: str | None) -> str:
     return backend
 
 
+# TODO: This is only used by vLLM but should move to vllm.py
```

Comment:
Any reason not to do that in this PR too?

Reply:
Because they are code refactors happening on the side of this PR that will pick it up :)